docs/test/code-generation-compilation-and-naming-conventions-in-microsoft-fakes.md

```diff
@@ -16,6 +16,7 @@ This article discusses options and issues in Fakes code generation and compilati
 **Requirements**
 
 -   Visual Studio Enterprise
+-   Only for .NET Framework projects. Currently, there is not support for .NET Framework projects that target .NET Standard. 
```

Comment:
@kendrahavens I find this sentence a tiny bit confusing because it says only for .NET Fx, but then not for some .NET Fx. To make it a little clearer, and to match the tense/structure of the first requirement, how about:

- a .NET Framework project

> [!NOTE]
> .NET Standard projects are not supported.

Reply:
That sounds good. I'll make this change in the other docs as well.